parsers/US_PJM.py

```diff
@@ -261,13 +261,15 @@ def get_exchange_data(interface, session: Session) -> list:
 
     flows = zip(actual_load, time_vals, strict=True)
 
-    arr_date = arrow.now("America/New_York").floor("day")
+    date = datetime.combine(datetime.now(TIMEZONE), time())  # truncate to day
 
     converted_flows = []
     for flow in flows:
-        arr_time = arrow.get(flow[1], "h:mm A")
-        arr_dt = arr_date.replace(hour=arr_time.hour, minute=arr_time.minute).datetime
-        converted_flow = (flow[0], arr_dt)
+        time_of_the_day = datetime.strptime(flow[1], "%I:%M %p").replace(
+            tzinfo=TIMEZONE
+        )
+        dt = date.replace(hour=time_of_the_day.hour, minute=time_of_the_day.minute)
+        converted_flow = (flow[0], dt)
```

Comment:
note: I couldn't test this because this function is not working for me (I think the API is gone)